freqtrade/persistence/trade_model.py

```diff
@@ -1017,6 +1068,7 @@ class Trade(_DECL_BASE, LocalTrade):
     open_trade_value = Column(Float)
     close_rate: Optional[float] = Column(Float)
     close_rate_requested = Column(Float)
+    realized_profit = Column(Float, default=0.0)
```

Comment:
For added columns - please make sure they're also part of the migration (and that the migration triggers).
otherwise this will: 
* fail on opening the first trade with an existing database
* fail when the next migratio is triggered (this column will not be migrated)